module/data/item/loot.mjs

```diff
@@ -16,7 +16,7 @@ export default class LootData extends SystemDataModel.mixin(
   /** @inheritdoc */
   static defineSchema() {
     return this.mergeSchema(super.defineSchema(), {
-      type: new ItemTypeField({}, {label: "DND5E.ItemLootType"})
+      type: new ItemTypeField({}, {subtype: false, baseItem: false, label: "DND5E.ItemLootType"})
```

Comment:
The system doesn't provide any subtypes for loot, but we do support it in the sheet if any were added via a module or world script.
https://github.com/foundryvtt/dnd5e/blob/master/module/applications/item/item-sheet.mjs#L150